.github/workflows/deployDev4.yml

```diff
@@ -0,0 +1,123 @@
+name: Deploy Dev4
```

Comment:
Not blocking. Not requesting a change. Potential future work.

Something we could do to dry this and possibly other deployment workflows up is to turn the bulk of these deployment tasks into a `workflow_call`, we'd still have to pass in the secrets and other inputs but it might give us a nice template that reduces the level of duplication. It could be used for all deployments without restricting our ability to customize deployments to specific environments when needed.

Reply:
I love the idea. I had thought about that when I was working on this, but didn't want to introduce extra scope creep. Ticket created for this: https://github.com/CDCgov/prime-simplereport/issues/3712

Reply:
:+1: , thanks for capturing that in a ticket.